README.md

```diff
@@ -84,7 +84,7 @@ As you see here - now the root element has the `scrollerElement` ref, but otherw
 #### Content sizes translation
 
 In some situations you may want to make the scrollbars block of variable sizes - just pass `translateContentSize*ToHolder` prop and component will automatically translate corresponding `contentElement`'s sizes to the `holderElement`.  
-If you are using default styles - it'll be handy to pass `compensateScrollbarsWidth={false}` props, to avoid infinite shrinking when it's not supposed to.  
+If you are using default styles - it'll be handy to pass `disableTracksWidthCompensation={false}` props, to avoid infinite shrinking when it's not supposed to.  
```

Comment:
Should be `disableTracksWidthCompensation` `disableTracksWidthCompensation={false}` is equal to absence of that prop